python/rmm/rmm/tests/test_rmm.py

```diff
@@ -1002,3 +1002,15 @@ def test_invalid_logging_level(level):
         rmm.set_flush_level(level)
     with pytest.raises(TypeError):
         rmm.should_log(level)
+
+
+def test_available_device_memory():
+    from rmm._lib.memory_resource import available_device_memory
```

Comment:
We should import this from `rmm.mr` so we're testing the public interface.
```suggestion
    from rmm.mr import available_device_memory
```